src/execution/utils.ts

```diff
@@ -78,13 +86,12 @@ type TransactionResult =
   | {
       type: 'intent'
       id: bigint
-      sourceChain?: number
+      sourceChains?: number[]
       targetChain: number
     }
 
 interface IntentData {
   type: 'intent'
-  hash: Hex
```

Comment:
Could put back but probably not needed for consumers

Reply:
nah dont think its needed